gems/pending/openstack/openstack_handle/handle.rb

```diff
@@ -74,6 +74,10 @@ def self.raw_connect(username, password, auth_url, service = "Compute", extra_op
         Fog.const_get(service).new(opts)
       end
     rescue Fog::Errors::NotFound => err
+      $fog_log.warn("MIQ(#{self.class.name}##{__method__}) "\
+                    "Service #{service} not available for openstack provider #{auth_url}")
+      $fog_log.warn(err.message)
+      $fog_log.warn(err.backtrace.join("\n"))
```

Comment:
Should all of this be wrapped in the `if err.message.include?("Could not find service")` from below?